### ISSUE TITLE
how to use cwd in feature map

### ISSUE THREAD

**Issue by wwjwy** [open-mmlab/mmrazor#45]
hi,
I notice that the cwd is used only in Logits map in segmentation task. But the paper also contains the cwd used in feature map, can you give me an example how to modify the config file?   https://github.com/open-mmlab/mmrazor/blob/b6c5fffe8f04284f7debd7a519b75ab6254ac617/configs/distill/cwd/cwd_cls_head_pspnet_r101_d8_pspnet_r18_d8_512x1024_cityscapes_80k.py#L94

**Comment by pppppM** [comment]
Because the model structure and training settings in CWD are quite different from mmseg, the CWD in mmrazor is not the same as the paper.
If you want to distill on the feature map, you can modify the corresponding name according to your model.
It should be `decode_head.bottleneck` in the PSPNet.
